api/test/nostd/unique_ptr_test.cc

```diff
@@ -122,7 +122,12 @@ TEST(UniquePtrTest, Reset)
 {
   bool was_destructed1;
   unique_ptr<A> ptr{new A{was_destructed1}};
-  bool was_destructed2;
+  /*
+    MSVC reports:
```

Comment:
nit: I'm not sure if we need this comment

Reply:
Removed